chapters/data/memory-security/drills/tasks/exec-shellcode/support/tests/Makefile

```
@@ -1,5 +1,5 @@
 SRC_PATH ?= ../src
-FULL_SRC_PATH = $(realpath $(SRC_PATH))
+FULL_SRC_PATH = "$(realpath $(SRC_PATH))"
 CPPFLAGS = -I. -I$(realpath $(SRC_PATH)) -I../utils
```

Shouldn't this use `FULL_SRC_PATH` for the same reason?
```suggestion
CPPFLAGS = -I. -I$(FULL_SRC_PATH) -I../utils
```